### PR TITLE
Ignore `uuid` dependencies

### PR DESCRIPTION
**Pull request by amoore108**
Overall changes
======
- Ignores `uuid` and `@types/uuid` package updates as these packages appear to affect Opera Mini negatively
- For ref - https://github.com/bbc/simorgh/pull/11801 and https://github.com/bbc/simorgh/pull/11840

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@ updates:
       - dependency-name: 'webpack'
         update-types: ['version-update:semver-major']
       - dependency-name: 'winston'
+      # Opera Mini unsupported packages
```
Comment (karinathomasbbc) [inline .github/dependabot.yml:19]: Could we also include a link to the failed PR from the past?

File: .github/dependabot.yml
Comment (karinathomasbbc) [inline .github/dependabot.yml:1]: While we're in here, could we add another couple of deps to skip for the meantime (I envisage us coming back to this list as part of a tech work in a JIRA dependency ticket... although if we don't keep on top of it, we might never revisit it 😭)

- emotion? (Given the current memory leak issue... or do we want to wait?)
- eslint (it's bust / requires ESM upgrade, and there are already a couple of tech tickets in the backlog)

Can we think of any others?

**Review by karinathomasbbc — APPROVED** [review]

**Review by Louis-Matsika — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (amoore108) [reply .github/dependabot.yml:19]: Good shout 👍

File: .github/dependabot.yml
Comment (amoore108) [reply .github/dependabot.yml:1]: I've added ignores for `eslint` major versions.

May leave `emotion` for now as it does seem _fairly_ active and may fix itself soon. If its becomes really noisy then we can add it in.

**Pull request merged**